src/style.rs

```diff
@@ -1020,6 +1021,14 @@ pub enum TextOverflow {
     Ellipsis,
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum Rotation {
+    Rotation0,
+    Rotation90,
+    Rotation180,
+    Rotation270,
+}
+
```

Comment:
This should have better naming like:
```suggestion
#[derive(Debug, Clone, Copy, PartialEq, Eq)]
pub enum Rotation {
    NoRotation,
    RotateRight,
    RotateUpsideDown,
    RotateLeft,
}
```

Reply:
although I don't feel like `rotate` is anyhow good

Reply:
Preferably it should somehow comply with how CSS defines this attributes

Reply:
ok, thanks for your advice. I will revise it right away